mirror/consensus_service.proto

```diff
@@ -21,6 +21,7 @@
 syntax = "proto3";
 
 package com.hedera.mirror.api.proto;
+option pbj_java_package = "com.hedera.hapi.mirror";
```

Comment:
Why are we putting mirror node API protos under a HAPI folder? I thought we said pbj would exclude mirror protos?

Reply:
Same question for the SDK.

Reply:
Good question, can probably remove from SDK and Mirror. I was mostly adding for completeness. They were going to be excluded in hedera-protobufs-java build. What is the preference, leave or remove?